woocommerce/woocommerce
# [Experimental] Refactor the interactivity dropdown to remove dependence on FormTokenField

### Changes proposed in this Pull Request:

* Remove FormTokenField html markup from Dropdown.php
* Remove FormTokenField from editor side rendering of collection filter blocks (including removing use of emotion to color text)
* Fix bugs where stock status filter did not support multi select dropdown
* Fix other small bugs or clean up (e.g. AssetsController bug not loading CSS for dropdown and checkbox list)

Closes #43158



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Test a couple of the filter blocks that have a dropdown mode (e.g. rating and stock status filter), both in single and multi select mode and see that they work as before.
2. Compare the styles to the old styles (simplest is to compare to trunk, could use JN for this purpose). I haven't made it pixel for pixel identical, but I feel that looks the same as the old one.
3. Also test overriding text color, see that the text color is respected in editor and front end for dropdown mode.




### Changelog entry




<details>

#### Significance




#### Type




#### Message

[Experimental] Refactor interactivity dropdown to remove FormTokenField. Also fix multi select for rating filter.

#### Comment 

</details>
